### ISSUE TITLE
Training subset

### ISSUE THREAD

**Issue by longkhanh-fam** [choijeongsoo/lip2speech-unit#14]
Hi,
Which subsets do you use to train in lrs2 and lrs3, only train-val subset or both train-val and pretrain? And what is the used ratio between subsets

**Comment by choijeongsoo** [comment]
Hi!

We used both trainval and pretrain subsets, excluding this [validation set](https://github.com/facebookresearch/av_hubert/blob/main/avhubert/preparation/data/lrs3-valid.id).

Regarding the ratio, approximately 1:10? For training LRS2 model, we used the full train and pretrain data from LRS2 dataset. For LRS3 model, we utilized unseen split of LRS3 dataset following [SVTS](https://arxiv.org/abs/2205.02058).

**Comment by longkhanh-fam** [comment]
Yes, thank you for your response.
So for training the LRS2, both pretrain and full training set of LRS2 are used. May I ask what exactly in the validation set of LRS2? Does it contain any pretrain samples like STVS?

**Comment by choijeongsoo** [comment]
We used official [validation subset](https://thor.robots.ox.ac.uk//lip_reading/data2/val.txt) of LRS2 dataset, and it doesn't contain any pretrain samples if I remember correctly.

**Comment by longkhanh-fam** [comment]
Thanks @choijeongsoo, really appreciate your support ! I will close the issue here.
